bom/pom.xml

```diff
@@ -12,15 +12,12 @@
     <name>Quarkus CXF Bill Of Material</name>
     <packaging>pom</packaging>
 
+    <properties>
+        <quarkus-cxf.version>1.5.0-SNAPSHOT</quarkus-cxf.version>
```

Comment:
Is this version replaced when cutting a release? I have to admit I'm not very familiar with the quarkiverse release process.

Reply:
Yes, the release-plugin takes care.